crates/runtime/src/component/dataset/acceleration.rs

```diff
@@ -0,0 +1,320 @@
+/*
+Copyright 2024 The Spice.ai OSS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+use data_components::util::column_reference::ColumnReference;
```

Comment:
This is where the acceleration module got moved too. Unfortunately Git/GitHub doesn't show the diff, but its fairly small - just the couple of references to `on_conflict`